package.json

```diff
@@ -28,6 +28,7 @@
     "eslint-plugin-prettier": "^2.2.0",
     "eslint-plugin-react": "^7.3.0",
     "flow-bin": "^0.42.0",
+    "gatsby-1-config-css-modules": "^1.0.3",
```

Comment:
This is what was missing to make the tests work https://github.com/gatsbyjs/gatsby/pull/1964/commits/d09cc11d3c5e29bb28ee9cff7807e95cb222f463 so you can remove this.